js/counties.js

```diff
@@ -92,7 +87,7 @@ function displayCountyData(error, activeCountyData) {
         return d.properties.GEOID;
       })
       .text(function(d) {
-        return d.properties.NAME;
+        return d.properties.COUNTY_LONG;
```

Comment:
i think we now have COUNTY and COUNTY_LONG available to us, and I decided that COUNTY looked better more often. i'll switch this if i can.